README.md

```diff
@@ -49,86 +51,99 @@ on, we suggest opening an issue detailing your ideas first.
-
-## Frequently Asked Questions
+All code should be formatted through `go fmt`. This tool will automatically
+format code for you. We recommend, however, that you write code in the proper
+style and use `go fmt` only to catch mistakes.
 
-#### Yay does not display colored output. How do I fix it?
-  Make sure you have the `Color` option in your `/etc/pacman.conf` [#123](https://github.com/Jguer/yay/issues/123)
+#### 5. Test
 
-#### Yay is not prompting to skip packages during sysupgrade (issue [#554](https://github.com/Jguer/yay/issues/554))
-  The default behavior was changed after [v8.918](https://github.com/Jguer/yay/releases/tag/v8.918)
-  (see: [3bdb534](https://github.com/Jguer/yay/commit/3bdb5343218d99d40f8a449b887348611f6bdbfc)).
-  To restore such behavior use `--combinedupgrade`. This can also be
-  permanently enabled by appending `--save`.
-  Note: this causes [native pacman](https://wiki.archlinux.org/index.php/AUR_helpers) to become partial.
+Run `make test` to test _yay_. This command will verify that the code is
+formatted correctly, run the code through `go vet`, and run unit tests.
 
-#### Sometimes diffs are printed to the terminal and other times they are paged via less. How do I fix this?
-  Yay uses `git diff` to display diffs, by default git tells less to not page
-  if the output can fit one terminal length. This can be overridden by
-  exporting your own flags `export LESS=SRX`.
+## Frequently Asked Questions
 
-#### Yay is not asking me to edit PKGBUILDS and I don't like diff menu! What do?
+#### _Yay_ does not display colored output. How do I fix it?
+  Make sure you have the `Color` option in your `/etc/pacman.conf`
+  (see issue [#123](https://github.com/Jguer/yay/issues/123)).
+
+#### _Yay_ is not prompting to skip packages during system upgrade.
+  The default behavior was changed after
+  [v8.918](https://github.com/Jguer/yay/releases/tag/v8.918)
+  (see [3bdb534](https://github.com/Jguer/yay/commit/3bdb5343218d99d40f8a449b887348611f6bdbfc)
+  and issue [#554](https://github.com/Jguer/yay/issues/554)).
+  To restore the package-skip behavior use `--combinedupgrade` (make
+  it permanent by appending `--save`). Note: skipping packages will leave your
+  system in a
+  [partially-upgraded state](https://wiki.archlinux.org/index.php/System_maintenance#Partial_upgrades_are_unsupported).
+
+#### Sometimes diffs are printed to the terminal, and other times they are paged via _less_. How do I fix this?
+  _Yay_ uses `git diff` to display diffs, which by default tells _less_ not to
+  page if the output can fit into one terminal length. This behavior can be
+  overridden by exporting your own flags (`export LESS=SRX`).
+
+#### _Yay_ is not asking me to edit PKGBUILDS, and I don't like the diff menu! What can I do?
   `yay --editmenu --nodiffmenu --save`
 
-#### Only act on AUR packages or only on repo packages?
-  `yay -{OPERATION} --aur`
+#### How can I tell _yay_ to act only on AUR packages, or only on repo packages?
```

Comment:
Wording is good, but I prefer both commands being on the same line as it was before.

Reply:
OK, reverted the spacing.